### ISSUE TITLE
Pokémon Black 2 error 20110

### ISSUE THREAD

**Issue by hfc2x** [barronwaffles/dwc_network_server_emulator#523]
I'm having that specific error on my legit Pokémon Black 2 game when trying to connect to the GTS. The funny part is that I have been able to connect before, and even earlier I managed to trade a Karrablast through the GTS, so there's nothing wrong with my configuration. I just won't be able to connect anymore, and I'll always get an error that the service has been discontinued.

![2](https://user-images.githubusercontent.com/8249996/60792803-5781b380-a134-11e9-9c28-94e4587d2fa1.png)

**Comment by barronwaffles** [comment]
Unsure sorry, just did a quick test and everything was working here.

**Comment by hfc2x** [comment]
Some other details, I'm using a Nintendo 3DS to connect, and it's an unsecured hotspot. I have no idea how any of that could influence anything, since Pokémon SoulSilver connects just fine with the same 3DS and network.

Did a quick test and deleted the system information, and tried making a new ID. The connection worked fine the first time, but all subsequent tries gave me the same error again and will not let Black 2 connect anymore. SoulSilver is still working fine. Just got a new Friend Code and all.

**Comment by mm201** [comment]
3DS uses different connection settings for DSi enhanced games. Check that the proper DNS is set in 3DS WiFi settings.

**Comment by hfc2x** [comment]
Huh, that's pretty interesting. Since Pokémon Black 2 says that the connection settings cannot be changed from within the game, I had gone into the System Settings and done that already (or so I thought). I hadn't configured a connection from the 3DS side, so I thought configuring the one from the list within the "Nintendo DS Connections" option (which is the same list you see if you configure it from the main menu in Gen IV games) would be enough. Turns out, DSi games actually use the 3DS connections lol.

It's pretty surprising, considering the first time it connected, it used one of the Nintendo DS connections I had already configured for use with this server, and it had no problem, but once I tried to reconnect, it tried doing so with a 3DS connection instead. I think it would be a good thing to document, don't you think?

EDIT: For the record, changed it from here, and now Pokémon Black 2 is connecting properly with no issues.

![3](https://user-images.githubusercontent.com/8249996/60841511-fbed0f80-a19f-11e9-8d37-8173519cd79c.png)
